<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-1GOPOLA3" failures="1" tests="2" name="Parallel Tests" time="17.652" errors="0" timestamp="29 May 2021 07:38:43 GMT">
  <testcase classname="Package1.NewTest" name="f" time="7.747"/>
  <testcase classname="Package1.NewTest" name="f1" time="17.6">
    <failure type="junit.framework.AssertionFailedError">
      <![CDATA[junit.framework.AssertionFailedError
at junit.framework.Assert.fail(Assert.java:47)
at junit.framework.Assert.assertTrue(Assert.java:20)
at junit.framework.Assert.assertTrue(Assert.java:27)
at Package1.NewTest.f1(NewTest.java:96)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:832)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- f1 -->
</testsuite> <!-- Parallel Tests -->
